docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2.2'
+
+# data volume for DB
+volumes:
+  dbstorage:
+
+# services (docker containers)
+services:
+  # MariaDB database
+  db:
+    image: mariadb:10.3
+    # Set max_allowed_packet to 256M
+    command: --max_allowed_packet=32505856
+    restart: always
+    container_name: variantstore_db
+    environment:
```

Comment:
We should also provide a different MYSQL user here (and password) if possible. This should also be provided in an `.env` file (see below).

Reply:
Definitely, good suggestion!

Reply:
Done -> docker-config.env